packages/url_launcher/url_launcher/pubspec.yaml

```diff
@@ -3,18 +3,24 @@ description: Flutter plugin for launching a URL on Android and iOS. Supports
   web, phone, SMS, and email schemes.
 author: Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher
-version: 5.2.7
+version: 5.3.0
 
 flutter:
   plugin:
-    androidPackage: io.flutter.plugins.urllauncher
-    iosPrefix: FLT
-    pluginClass: UrlLauncherPlugin
+    platforms:
+      android:
+        package: io.flutter.plugins.urllauncher
+        pluginClass: UrlLauncherPlugin
+      ios:
+        pluginClass: FLTUrlLauncherPlugin
```

Comment:
Nit: FLTURLLauncherPlugin (same for the filename). The old capitalization was wrong for ObjC, but there was no way to have it be different for ObjC and Java. Now that there is, it should be fixed.